haiku-apps/midikeyboard/midikeyboard-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An onscreen midi keyboard"
+DESCRIPTION="An awesome keyboard on the screen for playing music"
+HOMEPAGE="https://github.com/HaikuArchives/MidiKeyboard"
+COPYRIGHT="1991, 1989 Free Software Foundation"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+srcGitRev="25fad6b3f1207e5c7b62a8aa49b0a775fae71899"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="7fa674575ae0b0b963d152ea36f9802d8d1949bd4bfd9513c6fe1dc33712a891"
+SOURCE_DIR="MidiKeyboard-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	midikeyboard$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
```

Comment:
I think getting it to compile on x86 is out of scope for this task.
If we went the manual road, mkdir -p $appsDir, and copying the binary there, we could rename it to MidiKeyboard in one go. Which would please me... :)
Otherwise,  @dacianf , check the mkefile_engine to see what it expects for the target "install".